pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -248,6 +248,11 @@ type Suspension struct {
 	// Note: Can not co-exist with Dispatching which is used to suspend all.
 	// +optional
 	DispatchingOnClusters *SuspendClusters `json:"dispatchingOnClusters,omitempty"`
+
+	// Scheduling controls whether scheduling should be suspended.
+	// nil means not suspend, no default value, only accepts 'true'.
+	// scheduler will pause scheduling when value is true and resume scheduling when it's nil.
+	Scheduling *bool `json:"scheduling,omitempty"`
```

Comment:
It's not clear why we need flag field here, also the field "scheduling" looks confusing what does the admin really want to stop system from doing.
Whether the resourcebinding shall be created?

Reply:
The resourcebinding will be created but scheduler will not scheduler, because we need some admission check such as quota check, multi-tenancy prioritization, etc before schduler process resourcebinding.
Will add more comments: )

Reply:
Have decoupled in https://github.com/karmada-io/karmada/pull/5974.